README.md

```diff
@@ -417,12 +417,14 @@ Match directive is supported on SSH >= 5.x.
 
 - *Hiera example*:
 <pre>
+```yaml
```

Comment:
This messes up the formatting. Perhaps remove the pre tags and it will look right. I do like having the syntax highlighting. Thank you :)